tools/federation/src/test/resources/tests/rdf4jserver/config.ttl

```diff
@@ -9,7 +9,14 @@
    rep:repositoryImpl [
       rep:repositoryType "fedx:FedXRepository" ;
       fedx:fedxConfig "fedxConfig.prop" ;
-      fedx:dataConfig "dataConfig.ttl" ;
+      fedx:member [
+         fedx:store "ResolvableRepository" ;
+         fedx:repositoryName "endpoint1" 
+      ] ,
+      [
+         fedx:store "ResolvableRepository" ;
+         fedx:repositoryName "endpoint2" 
+      ]
```

Comment:
@jeenbroekstra can you please provide your feedback on this configuration file, and particularly define which namespace prefix we want to use in the future for the configuration settings?

This PR  currently contains the functional part and I would like to use the opportunity to finalize namespace prefixes here.

Current prefix:

fedx: <http://www.fluidops.com/config/fedx#>

Proposals:

fedx: <http://www.openrdf.org/config/federation#>.

Note: FedX is implemented as a Repository, basically extending the SailRepository (because it requires some parts of the API that are not available to the SAIL). The main functional part is still a SAIL.

The existing federation sail uses properties like: http://www.openrdf.org/config/sail/federation#member

Once we have this final, I will also start with the write-up of the documentation.

Reply:
For new vocabulary, I'd prefer using rdf4j.org rather than openrdf.org (which is a domain we no longer own). How about http://rdf4j.org/config/federation# ?

Reply:
Sounds good, will adjust the examples and provide a new commit